docs/hub/how-to-inference.md

```diff
@@ -0,0 +1,44 @@
+# How to integrate the Inference API in your library
+
+The Inference API provides faster inference for your hosted models. Integration with the Hub allows you to access the Inference API programatically. This guide will show you how to make calls to the Inference API from your library. For more detailed information, refer to the [Inference API documentation](https://api-inference.huggingface.co/docs/python/html/index.html).
+
+Begin by creating an instance of the `InferenceAPI` with a specific model repository ID:
+
+```python
+>>> from huggingface_hub.inference_api import InferenceApi
+>>> inference = InferenceApi(repo_id="bert-base-uncased", token=API_TOKEN)
+```
+
+The pipeline is determined from the metadata in the model card and configuration files. For example, when you use the [bert-base-uncased](https://huggingface.co/bert-base-uncased) model, the Inference API is able to automatically infer this model should be used for a `fill-mask` task.
```

Comment:
Are the configuration files explained somewhere else in the docs? If yes, we could link to them here so readers know what format is expected

Reply:
I didn't see it anywhere in the docs, but we could link them to examples. Are these just the `config.json` and `tokenizer_config.json` files in the repo?

Reply:
Added link to an example `config.json` file, so readers know what it looks like and what to expect.